applications/MeshingApplication/external_includes/trigen_droplet_refine.h

```diff
@@ -109,7 +109,7 @@ class TriGenDropletModeler
 
     //*******************************************************************************************
     //*******************************************************************************************
-    void ReGenerateMeshDROPLET(
+    void ReGenerateMeshDroplet(
```

Comment:
@elafnm This is the typo, the rest is just clean up

Reply:
Greetings dears

Agreed,

Best regards